Orange/widgets/unsupervised/owdbscan.py

```diff
@@ -180,8 +182,11 @@ def set_data(self, data):
             self.Outputs.annotated_data.send(None)
             self.plot.clear_plot()
             return
-
-        if self.data is None:
+        if self.data.X is None or self.data.X.shape[1] == 0:
```

Comment:
I think `self.data.X is None` could never be `True`.
`Table.X` is always a np.ndarray, but sometimes empty.

Reply:
I thought as much. Thanks for checking!